mscore/musescore.cpp

```diff
@@ -5502,8 +5502,11 @@ void MuseScore::cmd(QAction* a, const QString& cmd)
             setPlayRepeats(a->isChecked());
       else if (cmd == "pan")
             MScore::panPlayback = !MScore::panPlayback;
-      else if (cmd == "show-invisible")
+      else if (cmd == "show-invisible") {
             cs->setShowInvisible(a->isChecked());
+            cs->setLayoutAll();
```

Comment:
show-invisible does not change any layout, so setLayoutAll() should not be called

Reply:
I added this code because I need it to prevent another bug. When I disabled 'show invisible' and changed some note it led to layout. Layouting led to rebuilding bspTree. And BspTree didn't include such kind of elements (invisible and with disabled option 'show invisible'). When later I enabled 'show Invisible' unvisible elements didn't appear. Because it weren't included to BspTree. To see it there is some changing of score is needed. Because it again leads to rebuild BspTree.

Reply:
I'll tinking about solution without adding cs->setLayoutAll();

Reply:
done